contracts/Anchor.toml

```diff
@@ -21,6 +21,6 @@ test = "yarn run test"
 # TODO: add pubkeys
 
 [programs.localnet]
-ocr2 = "CF13pnKGJ1WJZeEgVAtFdUi4MMndXm9hneiHs8azUaZt"
-access_controller = "2F5NEkMnCRkmahEAcQfTQcZv1xtGgrWFfjENtTwHLuKg"
-store = "A7Jh2nb1hZHwqEofm4N8SXbKTj82rx7KUfjParQXUyMQ"
+ocr2 = "HW3ipKzeeduJq6f1NqRCw4doknMeWkfrM4WxobtG3o5v"
```

Comment:
Do we want to do this all the way to localnet? How are we going to share keys for CI localnet deployments which are currently here: https://github.com/smartcontractkit/chainlink-solana/tree/restructure-config/contracts/artifacts/localnet

Is it possible to do the initial deployment just by pointing to the pub key?

Reply:
localnet works specially in that Anchor inserts the programs into the genesis block at certain addresses (without needing the key). It's convenient because you can keep using the same addresses in tests. As you can see it's passing on the CI without the private keypairs

Reply:
That's nice, thanks for the explanation. Then additionally just delete those published localnet keys in this PR.